core-bundle/src/Resources/contao/library/Contao/Widget.php

```diff
@@ -1400,12 +1388,6 @@ public static function getAttributesFromDca($arrData, $strName, $varValue=null,
 			}
 		}
 
-		// Warn if someone uses the "encrypt" flag (see #8589)
```

Comment:
shouldn't we keep this and warn that encryption is not supported anymore?

Reply:
As I stated: It doesn't help anything at all. It does not work. Why would I have a warning for anything that does not work?

Reply:
because people might expect it to work? That's why we use deprecations, to tell people about changes in our code base …

Reply:
You use deprecation messages to inform people about something they shouldn't use anymore but it still works. Which is what we did for the last 3 years. But now it's not deprecated anymore. It doesn't work.

Reply:
May be it should be converted to an exception now instead, not just a warning.

Reply:
No. That's what you guys don't get. The exception was thrown ever since people used PHP 7.2. The `if (!\in_array('mcrypt', get_loaded_extensions()))` was always `true` so there is no way there's any installation out there still using `encrypt -> true` in their DCA.

Reply:
> so there is no way there's any installation out there still using encrypt -> true in their DCA.

Why would you assume that? If the extension or application only allowed or used older PHP versions so far and then updated to PHP >=7.3 and Contao >=4.11, there would be no indication anymore, that using `encrypt => true` does nothing.

Reply:
I’m with @fritzmg here. If someone uses `'encrypt' => true` (because of an update or an outdated tutorial/article) we should throw an exception.

Reply:
I've updated the PR to keep all the `encrpyt -> true` references which will now result in an exception right away.